code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -118,14 +118,12 @@
 	else
 		return ..()
 
-
-
 /obj/machinery/gibber/verb/eject()
 	set category = "Object"
-	set name = "empty gibber"
+	set name = "Empty gibber"
 	set src in oview(1)
 
-	if(usr.incapacitated())
+	if (usr.stat != CONSCIOUS || HAS_TRAIT(usr, TRAIT_HANDS_BLOCKED))
```

Comment:
Why remove the incapacitated check? Does it not fully meet the behavior you want?

Reply:
I removed it to make it like the food/slime processor's check.